tests_e2e/templates/execute-tests.yml

```diff
@@ -32,3 +32,14 @@ jobs:
           AZURE_CLIENT_SECRET: $(AZURE-CLIENT-SECRET)
           AZURE_TENANT_ID: $(AZURE-TENANT-ID)
           SUBSCRIPTION_ID: $(SUBSCRIPTION-ID)
+
+      - task: PublishTestResults@2
```

Comment:
I have general question regarding junit test results. Based on the current design when do we say test_case failed, is it if all the steps failed in test case or any of the step failed? 

Like dcr today do we show warning/failed status on test case if any of them failed, so that it's easy to navigate to that test case and dig the logs to see the errors.

Reply:
each scenario needs to define what conditions should be treated as an error (i.e. the test case must fail) or as warnings (i.e. the test passes, but there are some warning messages in the logs).

dcr is too limited in that it does not make this distinction and it is hard to know what is an actual failure without checking the logs in detail

Reply:
yes, but how do you pass this information to junit output, now junit output control by lisa and lisa determines the test case result.

Reply:
just pass or fail the test, not sure i get your question

Reply:
I understand pass or fail. It's straightforward. I'm curious about warning case If we make test case pass for warnings, then azure pipeline shows green and it's hard to know particular test case has warnings in log without going in each test case log.

Reply:
warning would be something that does not prevent a test from executing/succeeding. when a test fails. warnings can help determine why. 

not sure if there is a way to report them to the pipeline. but i am not sure we want to since they are only interesting when something fails

Reply:
if we consider one of step failure as warnings to continue execute rest of the steps, in those cases we still need to follow up why particular step as warning to make sure no regressions.

I'm just throwing point; this is something to think about in the grand scheme of things.

Reply:
not sure i understand what you are trying to achieve, but you can always reach out offline